src/aws-cpp-sdk-core/include/aws/core/utils/crypto/Cipher.h

```diff
@@ -16,7 +16,6 @@ namespace Aws
         namespace Crypto
         {
             static const size_t SYMMETRIC_KEY_LENGTH = 32;
-            static const size_t MIN_IV_LENGTH = 12;
```

Comment:
those are public constants? so removing them might be backwards incompatible?

Reply:
good eye